extensions/Cake.AssemblyInfoReflector.yml

```diff
@@ -8,4 +8,5 @@ Author: wexman
 Description: Cake addin for retrieving assembly info from an existing assembly.
 Categories:
 - assembly info
-TargetCakeVersion: 0.26.1
+TargetCakeVersion: 0.18.0
```

Comment:
Why the change here?

Reply:
Because previously we were analyzing the most recent version of this addin (which is 1.1.0-beta2) but the logic was changed to use the most recent "stable" version which is an older version:

![image](https://user-images.githubusercontent.com/112710/101052626-da3eb600-3554-11eb-81b9-c7a883cb9788.png)

Reply:
Ah, didn't know you had this also already done in this version 👍